blas/netlib/doc.go

```diff
@@ -85,9 +85,9 @@ See http://www.crest.iu.edu/research/mtl/reference/html/banded.html
 for more information
 
 */
-package blas // import "gonum.org/v1/netlib/blas"
+package netlib // import "gonum.org/v1/netlib/blas/netlib"
 
-// BUG(btracey): The cgo package is intrinsically dependent on the underlying C
+// BUG(btracey): The netlib package is intrinsically dependent on the underlying C
```

Comment:
I wonder if we should move this to the package documentation. It's not a bug, really, just a warning.

Shouldn't happen this PR, but I can open an issue if you agree.